cms/templates/home_page.html

```diff
@@ -10,9 +10,24 @@
 {% endblock %}
 
 {% block content %}
-  <h1>{{ page.title }}</h1>
-  <p>
-    Image:<br />
-    <img src="{% wagtail_img_src page.hero %}" alt="home" />
-  </p>
+  <style>
+    {% if page.hero %}
+      .hero-background {
+        background-image: url('{% wagtail_img_src page.hero%}');
+      }
+    {% endif %}
+  </style>
+
+    <div class="hero-content hero-background">
+      <div class="container hero-container">
```

Comment:
1. I think these class names are backwards. It makes more sense for the "container" element to contain the "content" element. This is a little confusing since we have a `container` class on that inner element, but that's just to apply the bootstrap styles
2. I don't think we need the new `hero-background` class. The styles can be applied directly to `.hero-container`
3. Did you try moving the `background-image` style definition into an inline style like I suggested?

```suggestion
  <div class="hero-container" style="background-image: url('{% wagtail_img_src page.hero%}')">
      <div class="container hero-content">
```

Reply:
Container brings padding (left, right) because of max-width that is always less than 100%, having said that our background will not fit on to the full screen (100% width) and we can't fill that padding (left, right) manually because when we switch from large to extra-large screens there is no upper limit so can't be sure that how much padding it would require in that case in order to make it fit (100%) to the screen size.